source/val/validate_memory.cpp

```diff
@@ -1594,7 +1596,8 @@ spv_result_t ValidateAccessChain(ValidationState_t& _,
       case spv::Op::OpTypeCooperativeMatrixNV:
       case spv::Op::OpTypeCooperativeMatrixKHR:
       case spv::Op::OpTypeArray:
-      case spv::Op::OpTypeRuntimeArray: {
+      case spv::Op::OpTypeRuntimeArray:
+      case spv::Op::OpTypeNodePayloadArrayAMDX: {
```

Comment:
This would be good to cover in memory test that access chain into a payload array is validated correctly.

Reply:
Test added.